data-rights-protocol.md

```diff
@@ -310,43 +367,48 @@ Note that these error states only represent *request errors*; workflow errors SH
-- [pem2jwk](https://github.com/mt-inside/pem2jwks) will accept an existing PEM-encoded key and convert it to JWK
-- [jwkcreator](https://russelldavies.github.io/jwk-creator/) web site MAY be used to convert PEM keys to JWKs which will not be used to exchange 3rd party identity and rights
-- [mkjwk.org](https://mkjwk.org/) web site MAY be used to generate JWKs which will not be used to exchange 3rd party identity and rights.
+We provide in OSIRAA an example [generating keys and signing
+requests](https://github.com/consumer-reports-digital-lab/osiraa/blob/main/drp_aa_mvp/data_rights_request/pynacl_validator_submit.py)
+using the PyNaCl implementation, as well as a [Django request
+handler](https://github.com/consumer-reports-digital-lab/osiraa/blob/main/drp_aa_mvp/data_rights_request/pynacl_validator.py)
+which does cryptographic verification of the requests (without the full semantic validation chain presented above).
 
 ### 3.08 Processing Extensions & "Expected By" dates
 
 When a Covered Business acknowledges receipt of a Data Rights Request and moves it in to `in_progress` state, the request's `expected_by` field SHOULD be populated based on either an estimate provided by the Covered Business or the deadline prescribed by the legal regime the request is submitted under. Consider, for example, [California's legal regime](https://transcend.io/laws/cpra/#section-15) prescribes up to 90 days extension so long as they are made within 45 days; If a request is extended, this request must also be extended with a `processing_details` field detailing a reason for the Request's extension to notify the consumer of this delay. The intent of the `processing_details` field is to add additional color to already-defined `state`/`reason` combinations. States which cannot be encoded without reaching for free-form text should be integrated in to the state transition table.
 
 When applying changes to Data Rights Requests in this fashion, the Privacy Infrastructure Provider SHALL attempt to notify the Authorized Agent using the [Data Rights Status Callback](#2041-post-status_callback-response).
 
+### 3.09 Request State Flow Diagram
+
+![Sequence Diagram showing Data Rights Request event flow](https://raw.githubusercontent.com/consumer-reports-digital-lab/data-rights-protocol/main/files/drp-1.0-sequence-diagram.svg)
+
 ## 4.0 Protocol Roadmap
 
-In its current implementation, DRP should not be used to process data of Users who are not involved in the implementers group. This protocol is undergoing significant technical design changes, security evaluation, and implementation work, which should preclude the transfer of arbitrary Users' data rights. 
+In its current implementation, DRP should not be used to process data of Users who are not involved in the implementers group. This protocol is undergoing significant technical design changes, security evaluation, and implementation work, which should preclude the transfer of arbitrary Users' data rights.
```

Comment:
I think also, and maybe more importantly, organizations who are not part of the implementers' group haven't signed on to its legal agreement and aren't bound by its standards.